buildspec.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023, OpenHW Group
+# SPDX-License-Identifier:Apache-2.0 WITH SHL-2.0
+#
+# Build-specification for the aws.yml workflow.
+
+version: 0.2
+
+phases:
+
+  build:
+    commands:
+      - chown -R florian:florian .
```

Comment:
I do not fully comprehend everything here, but are these scripts tied to a specific users account? should/could this be generalized?

Reply:
This is a good question.  The CI runs on a AWS VM instance that only OpenHW staff have access to.  It _could_ be any of us, but it cannot be "just anyone".  It probably makes sense to create a generic "openhw-ci-runner" account that has its access privileges tuned to the CI requirements (and no more).